src/api/http.go

```diff
@@ -44,6 +44,8 @@ const (
 	EndpointsWalletSeed = "WALLET_SEED"
 	// EndpointsDeprecatedWalletSpend endpoints implement the deprecated /api/v1/wallet/spend method
 	EndpointsDeprecatedWalletSpend = "DEPRECATED_WALLET_SPEND"
+	// EndpointsAll lists all of the above
+	EndpointsAll = "READ,STATUS,WALLET,WALLET_SEED,DEPRECATED_WALLET_SPEND"
```

Comment:
should be `[]string`, the comma separation is only relevant to the command line parser, this is the `api` package, although it does not belong in `api` since `api` does not use it, `all` is a command line parser idea